devolutions-gateway/src/recording.rs

```diff
@@ -228,6 +228,10 @@ enum RecordingManagerMessage {
         id: Uuid,
         session_must_be_recorded: bool,
     },
+    SubscribeToSessionEndNotification {
+        id: Uuid,
+        channel: oneshot::Sender<anyhow::Result<Arc<Notify>>>,
```

Comment:
suggestion: It’s fine to not return anything when there is an error.

```suggestion
        channel: oneshot::Sender<Arc<Notify>>,
```

See how we handle that for the Connect message:

https://github.com/Devolutions/devolutions-gateway/blob/31e4ae426f2de5fe8735a36f4d32072ebc7c2165/devolutions-gateway/src/recording.rs#L729-L736

Reply:
updated